package.json

```diff
@@ -21,33 +24,38 @@
     "ember-cli",
     "fragments"
   ],
-  "main": "ember-addon/index.js",
-  "ember-addon": {
-    "after": "ember-data"
+  "engines": {
+    "node": ">= 0.10.0"
   },
-  "files": [
-    "index.js",
-    "ember-addon"
-  ],
   "devDependencies": {
-    "broccoli-babel-transpiler": "^5.2.3",
-    "broccoli-defeatureify": "^1.0.0",
-    "broccoli-es3-safe-recast": "^2.0.0",
-    "broccoli-es6-module-transpiler": "^0.5.0",
-    "broccoli-funnel": "^0.2.3",
-    "broccoli-jshint": "^0.5.6",
-    "broccoli-merge-trees": "^0.2.1",
-    "broccoli-replace": "^0.3.1",
-    "broccoli-sourcemap-concat": "^0.4.4",
-    "broccoli-uglify-js": "^0.1.3",
-    "defeatureify": "^0.2.2",
-    "ember-cli": "1.13.12",
+    "broccoli-asset-rev": "^2.2.0",
+    "ember-ajax": "0.7.1",
+    "ember-cli": "2.4.2",
+    "ember-cli-app-version": "^1.0.0",
+    "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-htmlbars": "^1.0.1",
+    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
+    "ember-cli-inject-live-reload": "^1.3.1",
+    "ember-cli-pretender": "0.6.0",
+    "ember-cli-qunit": "^1.2.1",
     "ember-cli-release": "0.2.8",
-    "es6-module-transpiler": "^0.9.6",
-    "es6-module-transpiler-amd-formatter": "^0.3.0",
-    "es6-module-transpiler-package-resolver": "^1.0.1",
-    "publish": "^0.5.0",
-    "rsvp": "^3.0.21",
-    "testem": "0.8.3"
+    "ember-cli-sri": "^2.1.0",
+    "ember-cli-uglify": "^1.2.0",
+    "ember-data": "^2.4.0",
+    "ember-dev": "emberjs/ember-dev",
+    "ember-disable-prototype-extensions": "^1.1.0",
+    "ember-disable-proxy-controllers": "^1.0.1",
+    "ember-export-application-global": "^1.0.4",
+    "ember-load-initializers": "0.5.1",
+    "ember-resolver": "^2.0.3",
+    "ember-try": "^0.2.2",
+    "loader.js": "^4.0.0"
```

Comment:
@jakesjews I assume these packages were the result of running `ember init` in the root? It looks like a good chunk of them are unnecessary for our development, could you pare it down to only the ones we need?